packages/core/src/types.ts

```diff
@@ -114,6 +114,7 @@ export type NormalizedOverrideOutput = {
   useDates?: boolean;
   coerceTypes?: boolean; // deprecated
   useTypeOverInterfaces?: boolean;
+  useConstForSchemaDefinition?: boolean;
```

Comment:
Adding a prop is also required here.

https://github.com/orval-labs/orval/blob/master/packages/core/src/types.ts#L360

And please normalize `type` here:

https://github.com/orval-labs/orval/blob/master/packages/orval/src/utils/options.ts#L124

Then, `?` is not needed 👍 

```suggestion
  useConstForSchemaDefinition: boolean;
```

Reply:
Done